src/main/resources/META-INF/persistence.xml

```diff
@@ -1,24 +1,25 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<persistence xmlns="http://xmlns.jcp.org/xml/ns/persistence"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.1"
-  xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/persistence http://xmlns.jcp.org/xml/ns/persistence/persistence_2_1.xsd">
-  <persistence-unit name="CargoTrackerUnit">
-    <jta-data-source>java:app/jdbc/CargoTrackerDatabase</jta-data-source>
-    <shared-cache-mode>ALL</shared-cache-mode>
-    <properties>
-      <!-- The default behavior is not generating a schema. Schema generation is 
-        good for demos, RAD, development, etc but likely not a good idea for production. -->
-      <property
-        name="javax.persistence.schema-generation.database.action"
-        value="create" />
-      <!-- This will fail if it's not invoked on a blank database - this is fine. 
-        What this does is insert a single record that the application settings check can 
-        place a pessimistic lock on. -->
-      <property name="javax.persistence.sql-load-script-source"
-        value="META-INF/initital-data.sql" />
-      <!-- The default log level of info is best for production, want to see verbose 
-        output for development. -->
-      <property name="eclipselink.logging.level" value="FINEST" />
-    </properties>
-  </persistence-unit>
+<?xml version="1.0" encoding="UTF-8"?>
+<persistence xmlns="https://jakarta.ee/xml/ns/persistence"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="3.0"
+  xsi:schemaLocation="https://jakarta.ee/xml/ns/persistence https://jakarta.ee/xml/ns/persistence/persistence_3_0.xsd">
+  <persistence-unit name="CargoTrackerUnit">
```

Comment:
Headerr files has been changes only-:
1) xmlns is changed form JCP to Jakarta.EE
2) Shcema version updated from "2.1" to "3.0"